data/terms.json

```diff
@@ -7,6 +7,10 @@
     "term": "bitcoin",
     "meaning": "Satoshi Nakamoto, a pseudonym, developed the first decentralized, peer-to-peer digital money in 2009. Unlike government-issued currencies, Bitcoin boasts lower transaction fees than existing online payment options and is managed by a decentralized authority."
   },
+  {
+    "term": "satoshi",
+    "meaning": "the smallest denomination of BTC, equal to 0.00000001 bitcoin. Satoshis are named after Bitcoin’s pseudonymous creator, Satoshi Nakamoto."
```

Comment:
Thank you for your awesome contribution.

I have some feedback.

- Add the name of the term in the meaning eg. 
`Satoshi is the smallest denomination ....  ` not `the smallest demonination ...`